docs/tutorial/notebook/reading_a_config_file.ipynb

```diff
@@ -1,290 +1,694 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    !cat {cfg_file}
</pre><p>Use %pycat {cfg_file}</p><p>This will work on Windows as well, and will satisfy the style checker.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/540/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Sure!!